src/Maestro/Maestro.Data/Models/Build.cs

```diff
@@ -163,4 +163,12 @@ public class BuildDependency
         // To this build's repository and branch
         public double TimeToInclusionInMinutes { get; set; }
     }
+
+    public class GoalTime
```

Comment:
Please move the GoalTime class into a separate file. It isn't related to the Build table in any way.

Reply:
added a new class to capture goalTime